### PR TITLE
ZBUG-2592: Completely disable offline mode

### PR DESCRIPTION
**Pull request by lavegupta**
Instead of remove entire offline related code from app which may cause the problem if we touched essential code. We did following changes to disable offline mode.

1. Removed **offline mode** menu item from user menu. 
2. Removed _offlineSettingsListener method which was called by **offline mode** menu item. 
3. Translation key which was used in above.
4. Permanently set **isWebClientOfflineSupported** to **false**;

### CODE REVIEW

File: WebRoot/js/zimbraMail/core/ZmZimbraMail.js [hunk 1]
```diff
@@ -2365,11 +2365,6 @@ function(parent, parentElement, adminUrl) {
 
     menu.createSeparator();
 
-	if (!appCtxt.isExternalAccount() && appCtxt.get(ZmSetting.WEBCLIENT_OFFLINE_ENABLED)) {
-        mi = menu.createMenuItem("offlineSettings", {text: ZmMsg.offlineSettings});
-        mi.addSelectionListener(new AjxListener(this, this._offlineSettingsListener));
```
Comment (silentsakky) [inline WebRoot/js/zimbraMail/core/ZmZimbraMail.js:2370]: we should remove _offlineSettingsListener listener also and all ZmMsg keys used inside that

Comment (lavegupta) [reply WebRoot/js/zimbraMail/core/ZmZimbraMail.js:2370]: Removed

**Review by silentsakky — APPROVED** [review]

File: WebRoot/messages/ZmMsg.properties [hunk 2]
```diff
@@ -2450,9 +2450,7 @@ offlineHeapDump = Do we have your consent to upload heap dump of Zimbra Desktop\
 offlineMoveFolderError = Folder move failed, please try again later
 OfflineServerReachable = Connected to server. You are back online.
 OfflineServerNotReachable = Cannot connect to server. You are now offline.
-offlineSettings = Offline Mode
 offlineSignOutWarning = Signing out will also turn off the offline mode and clear the locally stored data.<br><br>Are you sure you want to continue?
```
Comment (miteshsavani810) [inline WebRoot/messages/ZmMsg.properties:2453]: Don't we need to remove `offlineSignOutWarning` ?

File: WebRoot/js/zimbraMail/core/ZmZimbraMail.js [hunk 3]
```diff
@@ -2655,18 +2650,6 @@ function(ev) {
 
 };
 
-ZmZimbraMail.prototype._offlineSettingsListener =
-function(ev) {
-    var dialog;
-    if (AjxEnv.isOfflineSupported) {
-        dialog = appCtxt.getOfflineSettingsDialog();
```
Comment (HimanshuSojitraSynacor) [inline WebRoot/js/zimbraMail/core/ZmZimbraMail.js:2662]: can we remove the related code?

File: WebRoot/messages/ZmMsg.properties (hunk 2 above)
Comment (lavegupta) [reply WebRoot/messages/ZmMsg.properties:2453]: https://github.com/Zimbra/zm-web-client/pull/868#issue-2248326894

File: WebRoot/js/zimbraMail/core/ZmZimbraMail.js (hunk 3 above)
Comment (lavegupta) [reply WebRoot/js/zimbraMail/core/ZmZimbraMail.js:2662]: https://github.com/Zimbra/zm-web-client/pull/868#issue-2248326894

Comment (lavegupta) [reply WebRoot/js/zimbraMail/core/ZmZimbraMail.js:2662]: https://github.com/Zimbra/zm-web-client/pull/868#issue-2248326894

**Review by miteshsavani810 — APPROVED** [review]

**Review by HimanshuSojitraSynacor — APPROVED** [review]

**Pull request merged**